LayoutTests/accessibility/mac/text-operation/text-operation-capitalize.html

```diff
@@ -0,0 +1,111 @@
+        });
+        output += expect("operationResult.length", "1");
+        output += expect("operationResult[0]", "'Brown Fo'");
+        output += expect("text.stringValue.replace(/\\s/g, ' ')", "'AXValue: The quick Brown Fox jumps over the lazy dog.'");
+
+        // Validate that text is transformed to capitalized as expected (with smart replacement).
+        await waitForNotification(text, "AXValueChanged", async () => {
+            operationResult = text.performTextOperation("Capitalize", [await selectElementTextById("target2")], null, /* smart replace */ true);
+        });
+        output += expect("operationResult.length", "1");
+        output += expect("operationResult[0]", "'Er The La'");
+        output += expect("text.stringValue.replace(/\\s/g, ' ')", "'AXValue: The quick Brown Fox jumps ov Er The La zy dog.'");
+
+        // Validate that multiple ranges of text are transformed to capitalized as expected (without smart replacement).
+        text = accessibilityController.accessibleElementById("text2");
+        var markers = [await getMarkerRangeInElementById("text2", 42, 46), await getMarkerRangeInElementById("text2", 30, 37), await getMarkerRangeInElementById("text2", 9, 20)];
+        await waitForNotification(text, "AXValueChanged", async () => {
+            operationResult = text.performTextOperation("Capitalize", markers, null, /* smart replace */ false);
+        });
+        output += expect("operationResult.length", "3");
+        output += expect("operationResult[0]", "'Lazy'");
+        output += expect("operationResult[1]", "'Ps Over'");
+        output += expect("operationResult[2]", "'E Quick Bro'");
+        output += expect("text.stringValue.replace(/\\s/g, ' ')", "'AXValue: TEXT2: ThE Quick Brown fox jumPs Over the Lazy dog.'");
+
+        // Validate that multiple ranges of text are transformed to capitalized as expected (with smart replacement).
+        // The following test is EXPECTED TO FAIL. See https://bugs.webkit.org/show_bug.cgi?id=278928
+        text = accessibilityController.accessibleElementById("text3");
+        markers = [await getMarkerRangeInElementById("text3", 42, 46), await getMarkerRangeInElementById("text3", 30, 37), await getMarkerRangeInElementById("text3", 9, 20)];
+        await waitForNotification(text, "AXValueChanged", async () => {
+            operationResult = text.performTextOperation("Capitalize", markers, null, /* smart replace */ true);
+        });
+        output += expect("operationResult.length", "3");
+        output += expect("operationResult[0]", "'Lazy'");
+        output += expect("operationResult[1]", "'Ps Over'");
+        output += expect("operationResult[2]", "'E Quick Bro'");
+        output += expect("text.stringValue.replace(/\\s/g, ' ')", "'AXValue: TEXT3: Th E Quick Bro wn fox jum Ps Over the Lazy dog.'");
+
+        document.getElementById("test-content").remove();
```

Comment:
It seems like a hidden element still appears in the expected text, so I removed it altogether.